src/intro.md

```diff
@@ -10,11 +10,11 @@ This is the official “LinuxBoot Book” for the LinuxBoot project. The book:
 
 ## What is LinuxBoot?
 
-LinuxBoot is based on the idea of replacing proprietary or corporate-driven
-late-stage boot [firmware](./glossary.md) with the Linux kernel and a
-community-based userland. That idea grew into a project that over the years
-includes various initiatives with the overarching goal of moving from obscure
-and complex firmware to simpler and open source firmware.
+LinuxBoot is the idea of replacing proprietary or corporate-driven late-stage
```

Comment:
should we say "started with" to make clear that this is not the status quo, so people want to keep reading?

Reply:
"LinuxBoot started with the idea" doesn't define what LinuxBoot is. With the many different meanings I want to clearly define it at the start. What is LinuxBoot? LinuxBoot is ...

Reply:
See also https://www.linuxboot.org